cypress/e2e/tests/board.cy.js

```diff
@@ -0,0 +1,16 @@
+import { it } from "mocha";
+import { BoardHelper } from "../pages/board/board.helper";
+import { LoginConstants } from "../pages/login/login.constants";
+import { LoginHelper } from "../pages/login/login.helper";
+
+describe('Verify that it is possible to create a new board.', () => {
+    const username = Cypress.env('CYPRESS_USERNAME');
+    const password = Cypress.env('CYPRESS_PASSWORD');
```

Comment:
```suggestion
    beforeEach(() => {
        const username = Cypress.env('CYPRESS_USERNAME');
        const password = Cypress.env('CYPRESS_PASSWORD');
        LoginHelper.login(username, password);
    });
```
//te sujiero usar before each para que las constantes te funcionen para cada test